dasharo-security/measured-boot.robot

```diff
@@ -18,25 +18,376 @@ Resource            ../keys.robot
+    ${tpm2_eventlog}=    Execute Command In Terminal
+    ...    tpm2_eventlog /sys/kernel/security/tpm0/binary_bios_measurements
+    Should Not Contain    ${tpm2_eventlog}    ERROR: Unable to run tpm2_eventlog
+    FOR    ${algo}    IN    sha1    sha256
+        ${eventlog_pcrs}=    Get PCRs From Eventlog    ${tpm2_eventlog}    ${algo}
+        FOR    ${pcr_element}    IN    @{eventlog_pcrs}
+            ${pcr}    ${hash}=    Split String    ${pcr_element}    separator=:
+            ${sha_hash}=    Execute Command In Terminal
+            ...    cat /sys/class/tpm/tpm0/pcr-${algo}/${pcr}
+            Should Contain    ${hash}    ${sha_hash}    ignore_case=${TRUE}
+        END
+    END
+
+MBO003.001 Changing Secure Boot certificate changes only PCR-7
+    [Documentation]    Check if changes to Secure Boot certificates influence
+    ...    PCR-7 value and only PCR-7
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    Tests in firmware are not supported
+    Skip If    not ${SECURE_BOOT_SUPPORT}    Secure Boot tests are not supported
+    Restore Secure Boot Defaults
+    ${sb_menu}=    Reenter Menu And Return Construction
+    Disable Secure Boot    ${sb_menu}
+    Save Changes And Reset
+
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    ${default_hashes}=    Get PCRs State From Linux
+
+    Power On
+    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
+    ${sb_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
+    ${sb_menu}=    Enter Submenu From Snapshot And Return Construction    ${sb_menu}    DBX Options
+    ${sb_menu}=    Enter Submenu From Snapshot And Return Construction    ${sb_menu}    Delete Signature
+    Enter Submenu From Snapshot    ${sb_menu}    Delete All Signature List
+    Read From Terminal Until    Press 'Y' to delete signature list
+    Write Into Terminal    Y
+    Sleep    1s
+    Save Changes And Reset
+
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    FOR    ${pcr_hash}    IN    @{default_hashes}
+        ${pcr}    ${hash}=    Split String    ${pcr_hash}    separator=:
+        ${new_hash}=    Execute Command In Terminal    cat ${pcr}
+        IF    '/7' in '${pcr}'
+            Should Not Be Equal    ${hash}    ${new_hash}
+        ELSE
+            Should Be Equal    ${hash}    ${new_hash}
+        END
+    END
+
+MBO004.001 Changing Dasharo network boot settings changes only PCR-1
+    [Documentation]    Check if changes to Dasharo security settings influence PCR-1
+    ...    value and only PCR-1
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    Tests in firmware are not supported
+    Skip If    not ${DASHARO_NETWORKING_MENU_SUPPORT}    Tests in Dasharo Networking Menu are not supported
+    Power On
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    @{default_hashes}=    Get PCRs State From Linux
```

Comment:
Here and at the start of other tests.  Is this really default state or maybe the current one?  If it must be default, I guess `Get Default PCRs State` should be used instead.

Reply:
Changed in [d4ff4f0](https://github.com/Dasharo/open-source-firmware-validation/pull/326/commits/d4ff4f0868ebc8c13979ac1e53b879138aa269ad) to `hashes_before_changes` as `current_hashes` wouldn't be current couple lines later.